src/pytest/test.py

```diff
@@ -1405,98 +1405,6 @@ def testInfoCommand(self):
                 for option in filter(None, combo):
                     self.assertTrue(option in opts)
 
-    def aofTestCommon(self, reloadfn):
```

Comment:
This means we have no test for aof with rdb preamble. How do we test that scenario?

Reply:
I've restored them